logs/windows/3.9.13/protocol_4/test_0.json

```diff
@@ -0,0 +1 @@
+{"a": 1, "b": 5, "c": 3}
```

Comment:
I do queston why we have these files pushed. While one can argue it's for testing purposes, I don't feel like that was the intent here. We should add some kind of gitignore here i feel, since these are files that will change often.

Reply:
yes we can do that. we had it previously but after the file structure change they stopped working

Reply:
Made a fix, ignore them now.